core/keyserver/keyserver_test.go

```diff
@@ -56,9 +62,14 @@ func TestListEntryHistory(t *testing.T) {
 		// Test case setup.
 		c := &fakeCommitter{make(map[string]*tpb.Committed)}
 		st := &fakeSparseHist{make(map[int64][]byte)}
-		a := &fakeAppender{0, 0}
-		srv := New(c, fakeQueue{}, st, a, fakePrivateKey{}, fakeMutator{}, authentication.NewFake(), fakeFactory{})
-		if err := addProfiles(profileCount, c, st, a); err != nil {
+		fakeLog := fake.NewFakeTrillianClient()
+		admin := admin.NewStatic()
+		if err := admin.AddLog(logID, fakeLog); err != nil {
+			t.Fatalf("failed to add log to admin: %v", err)
+		}
+		sths := appender.NewTrillian(admin)
```

Comment:
Can you please mock `sths` similar to what we did with `fakeAppender`? I don't think testing key server depends on having a real `sths` object. Therefore, a fake `sths` does not need to do anything other than implementing the interface.

Same thing if this applies to other tests in this file.